src/views/User/User.tsx

```diff
@@ -177,7 +259,7 @@ export class User extends React.PureComponent<UserProperties, any> {
     }
 
     isSpecialUser() {
-        if (this.state.user.supporter || this.state.user.is_moderator || this.state.user.is_superUser) {
```

Comment:
I don't know what a "super user" is, but it seemed pretty clear this was a typo.  I did check the API call to verify this should be lowercase, but not sure where to verify in the UI.

Reply:
It should indeed be lowercase, thanks for the fix!. And FTR a superuser is matburt or myself

Reply:
Oh got it! That would explain the :spinning_smiley: OGS Developer :spinning_smiley: